lib/routes/admin/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+
+module.exports = function(mediator) {
+  const router = express.Router();
+
+  // TODO: Remove this endpoint when not needed
+  if (process.env.NODE_ENV === 'development') {
+    /**
+     * Asks for data to be reset to the default list of users
+     * This is intended only for demonstration/debugging purposes
+     * and by default only runs on NODE_ENV === 'development'
+     */
+    router.route('/data').delete(function(req, res) {
```

Comment:
Might be good to have this as a separate file with a clear warning at the top so it can easily be removed when not required.